internal/serverinstall/docker.go

```diff
@@ -226,7 +226,7 @@ func (i *DockerInstaller) Install(
 		Image:        i.config.serverImage,
 		ExposedPorts: nat.PortSet{npGRPC: struct{}{}, npHTTP: struct{}{}},
 		Env:          []string{"PORT=" + grpcPort},
-		Cmd:          []string{"server", "run", "-accept-tos", "-vv", "-db=/data/data.db", fmt.Sprintf("-listen-grpc=0.0.0.0:%s", grpcPort), fmt.Sprintf("-listen-http=0.0.0.0:%s", httpPort)},
+		Cmd:          []string{"server", "run", "-accept-tos", "-vv", "-db=/data/data.db", fmt.Sprintf("-listen-grpc=0.0.0.0:%s", grpcPort), fmt.Sprintf("-listen-http=0.0.0.0:%s", httpPort), opts.ServerRunFlags},
```

Comment:
I don't think this does what you expect, if you specify `-run-flags="-vv -db=/other.db"` then this will pass `-vv -db=/other.db` as a _single argument_. The exec API for all these platforms expects the arguments to be pre-split (which is usually done by a shell). Every platform has this issue.

Splitting by space is unsafe cause you also have to handle quotes and escapes and so on.

(see my PR comment but I recommend not accepting strings like this)